frontend/templates/lobby.html

```diff
@@ -1610,6 +1610,31 @@
 
             return circleData;
         }
+        
+        let sizeSlot = 8;
```

Comment:
There should already be a variable with the current size. It's called lineWidthUnscaled If I remember correctly.